src/components/AssignedMembers/hooks/useAssignedUsers/useAssignedUsers.js

```diff
@@ -0,0 +1,71 @@
+import { useQuery } from 'react-query';
+
+import {
+  useNamespace,
+  useOkapiKy,
+} from '@folio/stripes/core';
+
+import { MAX_RECORDS } from '../../../../constants';
+import { GROUPS_API, PERMISSIONS_API, USERS_API } from '../../constants';
+import { batchRequest, buildQueryByIds } from './utils';
+
+const useAssignedUsers = ({ grantedToIds = [], permissionSetId }, options = {}) => {
+  const ky = useOkapiKy();
+  const [namespace] = useNamespace({ key: 'get-assigned-users' });
+  const {
+    isLoading,
+    data = [],
+  } = useQuery(
+    [namespace, permissionSetId],
+    async ({ signal }) => {
+      const permissionUsersResponse = await batchRequest(
+        ({ params: searchParams }) => ky
```

Comment:
`X-Okapi-Tenant` header should be bound to provided `tenantId` or to the current (from stripes.okapi) if it is not provided.